ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-adaptive-virtual-topology.j2

```diff
@@ -28,6 +28,48 @@ Topology role: {{ router_adaptive_virtual_topology.topology_role }}
 {%         endif %}
 {%     endif %}
 
+{%     if router_adaptive_virtual_topology.profiles | length > 0 %}
+#### AVT Profiles
+
+| Profile name | Load balance policy | Internet exit policy |
+| ------------ | ------------------- | -------------------- |
+{%         for profile in router_adaptive_virtual_topology.profiles | arista.avd.natural_sort('name') %}
+| {{ profile.name }} | {{ profile.load_balance_policy | arista.avd.default("-") }} | {{ profile.internet_exit_policy | arista.avd.default("-") }} |
+{%         endfor %}
+{%     endif %}
+
+{%     if router_adaptive_virtual_topology.policies | length > 0 %}
```

Comment:
we have a kind of strict way of where we put the `newlines` in order to not render too many world.
Also I checked what we do in other doc templates and we check for the variable definition (if it does not exist actually checking the length will fail)
```suggestion
{%     if router_adaptive_virtual_topology.policies is arista.avd.defined %}

```